.gitignore

```diff
@@ -1,3 +1,4 @@
 *.sublime-project
 *.sublime-workspace
 *.swp
+.idea
```

Comment:
Should this (and other personal dev tools things already in .gitignore) be moved out of this repository? Typically these would belong in a `.gitignore_global` file. In git this is the configuration option called `excludesfile`.

https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000154070-How-to-gitignore-idea-files?page=1#community_comment_115000258150

Reply:
Hi Ian, I've removed this entry. You're quite right, it was just a reflex action for me to add it. I've left the others in there for now as I don't want to break anyone else's flow - it seems out of scope for this RFC, but I'd be happy to sponsor another pull request removing them.